controllers/hacker.controller.js

```diff
@@ -92,8 +92,19 @@ function sentDayOfEmail(req, res) {
     });
 }
 
+function completedBatchUpdate(req, res) {
+    return res.status(200).json({
+        message: Constants.Success.HACKER_UPDATE_BATCH,
+        data: {
+            success_ids: req.body.ids,
+            errors: req.errors || []
+        }
+    });
+}
+
 module.exports = {
     updatedHacker: updatedHacker,
+    completedBatchUpdate: completedBatchUpdate,
```

Comment:
nit: don't think this fits with how the other functions are named here. maybe `batchUpdatedHackers` but that sounds like a mouthful idk